### ISSUE TITLE
5.15.40-xanmod1-tt: ../kernel/sched/bs_nohz.h:712:17: error: 'cfs_rq' undeclared (first use in this function)

### ISSUE THREAD

**Issue by Atemu** [xanmod/linux#265]
```
...
  CC      kernel/power/qos.o
  CC      kernel/sched/idle.o
  AR      init/built-in.a
  CC      kernel/sched/bs.o
  CC      arch/x86/realmode/init.o
  CC      arch/x86/entry/vdso/vgetcpu.o
  CC      arch/x86/events/amd/iommu.o
  AS      arch/x86/realmode/rm/header.o
  AS      arch/x86/realmode/rm/trampoline_64.o
  AS      arch/x86/realmode/rm/stack.o
  AS      arch/x86/realmode/rm/reboot.o
  AS      arch/x86/realmode/rm/wakeup_asm.o
  CC      arch/x86/realmode/rm/wakemain.o
  CC      arch/x86/hyperv/nested.o
  AS      arch/x86/entry/vdso/vsgx.o
  CC      arch/x86/events/zhaoxin/core.o
  CC      kernel/locking/semaphore.o
  CC      arch/x86/events/intel/pt.o
  HOSTCC  arch/x86/entry/vdso/vdso2c
  CC      arch/x86/realmode/rm/video-mode.o
  CC      arch/x86/xen/grant-table.o
  AS      arch/x86/realmode/rm/copy.o
  CC      arch/x86/kernel/fpu/init.o
  AS      arch/x86/realmode/rm/bioscall.o
  CC      arch/x86/kernel/cpu/mce/core.o
  CC      arch/x86/realmode/rm/regs.o
  CC      kernel/power/main.o
  CC      arch/x86/realmode/rm/video-vga.o
In file included from ../kernel/sched/bs.c:1587:
../kernel/sched/bs_nohz.h: In function 'nohz_try_pull_from_candidate':
../kernel/sched/bs_nohz.h:712:17: error: 'cfs_rq' undeclared (first use in this function)
  712 |                 cfs_rq = &rq->cfs;
      |                 ^~~~~~
../kernel/sched/bs_nohz.h:712:17: note: each undeclared identifier is reported only once for each function it appears in
  LDS     arch/x86/entry/vdso/vdso32/vdso32.lds
  CC [M]  arch/x86/events/amd/power.o
  CC      arch/x86/realmode/rm/video-vesa.o
  AS      arch/x86/entry/vdso/vdso32/note.o
make[2]: *** [../scripts/Makefile.build:288: kernel/sched/bs.o] Error 1
make[2]: *** Waiting for unfinished jobs....
...
```

**Comment by xanmod** [comment]
Attach your kernel config here

**Comment by Atemu** [comment]
Mostly [this](https://github.com/NixOS/nixpkgs/blob/034257035fcab05260b885670430b50e3505c807/pkgs/os-specific/linux/kernel/xanmod-kernels.nix#L27-L68). (If you've got suggestions on that, we'd love to hear them.)

I'm not near my Linux machine atm to paste the full config but the important one is `CONFIG_NO_HZ_FULL` which is set.

It looks like a simple oversight where the declaration was deleted from the general case because it's not needed anymore but is actually still needed in the `NO_HZ_FULL` guarded code.

Fix is in #266.
